doc/releases/changelog-0.23.0.md

```diff
@@ -489,6 +497,14 @@
   1
   ```
 
+* The `qml.finite_diff` function has been deleted. Please use `qml.gradients.finite_diff` to compute
+  the gradient of tapes of QNodes. Otherwise, manual implementation is required.
+  [(#2464)](https://github.com/PennyLaneAI/pennylane/pull/2464)
+
+* The `get_unitary_matrix` transform has been removed, users should use
+  `qml.matrix` instead.
+  [(#2457)](https://github.com/PennyLaneAI/pennylane/pull/2457)
+
 * The `update_stepsize` method is being deleted from `GradientDescentOptimizer` and its child
```

Comment:
```suggestion
* The `update_stepsize` method has been removed from `GradientDescentOptimizer` and its child
```

In case it happens elsewhere and I didn't spot it, lets make sure that:

- We are using the past tense, not present tense
- We use 'removed' instead of 'deleted'.

Reply:
:+1: For my understanding: what would be the difference between removed and deleted?

Reply:
Oh it just sounds more formal in my head!